tests/acceptance/features/apiSpaces/etagPropagation.feature

```diff
@@ -0,0 +1,64 @@
+@api
+Feature: check etag propagation after different file alterations
+
+  Scenario: as share receiver copying a file inside a folder changes its etag for all collaborators
+    Given user "Alice" has been created with default attributes and without skeleton files
+    And user "Brian" has been created with default attributes and without skeleton files
+    And using spaces DAV path
```

Comment:
do we need this step on ocis?

Reply:
yes, otherwise the default dav version will be used.